src/amo/utils/index.js

```diff
@@ -42,25 +42,32 @@ export function getAddonURL(slug: string): string {
   return `/addon/${slug}/`;
 }
 
+// Note that this also excepts an object with any number of query params which
```

Comment:
Do we need such a comment? Maybe because `otherQueryParams` is an object and not the list of all extra parameters passed to the function. If we keep this approach (because rest syntax wouldn't work for instance), maybe this parameter could be renamed to `overrideQueryParams`?

Reply:
Yeah, I can rename it.